bazel/test_constants.bzl

```diff
@@ -42,3 +45,9 @@ TAG_NON_SANITY_TEST = ["nonsanity_test"] + TAG_MANUAL
 TAG_NON_SANITY_TEST_SETUP = ["nonsanity_setup"] + TAG_MANUAL
 TAG_NON_SANITY_TEST_TEARDOWN = ["nonsanity_teardown"] + TAG_MANUAL
 TAG_TRAFFIC_SERVER_TEST = ["traffic_server_test"]
+
+# Tag for utility scripts that are used in the Magma VM.
```

Comment:
Is this true for all of these scripts, can they only be used in the magma vm? And not in the deb/test/trfserver vms?

Reply:
I think they were all created for the Magma VM, but most likely some of them would work in other environments too.